linkerd/app/integration/src/lib.rs

```diff
@@ -3,6 +3,8 @@
 #![deny(warnings, rust_2018_idioms)]
 #![recursion_limit = "256"]
 #![type_length_limit = "16289823"]
+// It's not clear where this originates.
```

Comment:
🙃 someday we really need to at least break up the giant macros into per-test ones, this has gotten pretty unsustainable...